packages/console/src/components/Sidebar/index.tsx

```diff
@@ -24,7 +24,7 @@ const Sidebar = () => {
               key={title}
               titleKey={title}
               icon={<Icon />}
-              isActive={location.pathname === getPath(title)}
+              isActive={location.pathname.startsWith(getPath(title))}
```

Comment:
Why use `startsWith`?

Reply:
considering subpath:

`/connectors`
`/connectors/github`

this 2 path should all match "active".

Reply:
What if there's another route, say `/connectors/google`? Would it also be matched as active?

Reply:
of course, it SHOULD be active.